### ISSUE TITLE
[Bug]: Guidelines page accessibility issues

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ods-ios#554]
### Prerequisites

- [X] I have [searched the backlog](https://github.com/Orange-OpenSource/ods-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14 Plus

### OS Version

17.1.1

### App version

v0.16.0

### Describe the issue

This list is about the defaut page displayed in app : "Guidelines"

- [x] #657
**Recommandation : do not use orange for text on light backgrounds. For mobile use, recommanded text contrast is 7:1**

- [x] #658

- [x] #659
![image](https://github.com/Orange-OpenSource/ods-ios/assets/105045667/47381f0d-54fc-48ba-8b4d-79c98ee7f7ed)
**Recommandation : add a second way to transmist information**, like following example :
![image](https://github.com/Orange-OpenSource/ods-ios/assets/105045667/064a4e88-87dd-47ab-9d75-d2670dcebab3)

- [x] svg vocalization #702

- [x] keyboard focus barely visible on the 3 main buttons
![image](https://github.com/Orange-OpenSource/ods-ios/assets/105045667/fe967083-9ff7-476c-8de8-1cce6c65b3d4)
**Recommandation: If possible, add an outline to make the focus more visible or reinforce the focus with darker orange.** Maybe with this kind of code:
```
override open func accessibilityElementDidBecomeFocused() {
        backgroundColor = .focused
    }
```
- [x] voice over grouping: when user changes the colour theme, the description of the warning popup is vocalized with title only.
**Recommandation: Group popup title and description so the user has the whole information at once.**

- [x] landscape mode : landscape mode from guidelines page leads to open typography automatically
**Recommandation: no content change on landscape mode

- [x] tap cancellation not possible on colour palette icon. User should be able to cancel the tap by moving the finger out of the button
**Recommandation : Don't activate button on down action**

- [x] #683
**Recommandation: Use announcements (notifications) to force the title page vocalization - To be done on each main page Guidelines, Components, Modules & About**

### Expected Behavior

Page 100% accessible (updated status 19/12/2023: 71%)

**Comment by ludovic35** [comment]
Tu avoid vocalisation of text in image do:
group bouton + image via accessibilityElement(children: .contain)

**Comment by ludovic35** [comment]
it does not work, need more investigation

**Comment by pya35** [comment]
landscape mode : landscape mode from guidelines page leads to open typography automatically seems to be fixed on last release

**Comment by pya35** [comment]
keyboard focus visibility can be solved with high contrast in Full Keyboard Access settings, no fix needed

**Comment by pylapp** [comment]
@pya35 About your comment "_information transmitted only by color : selected tab differs only by orange colour._", it is the native iOS behavior.
Did you look on some a11Y settings to get workarounds?

**Comment by pylapp** [comment]
@pya35 I understand the needs behind your case _"tap cancellation not possible on colour palette icon. User should be able to cancel the tap by moving the finger out of the button"_, however it is the native behaviour of iOS for such pickers: the content is displayed when the component is pressed, it's slightly different than buttons for example.
